compiler/passes/src/static_single_assignment/rename_statement.rs

```diff
@@ -369,6 +370,23 @@ impl StatementConsumer for StaticSingleAssigner<'_> {
                     id: input.id,
                 }))
             }
+            Expression::Access(AccessExpression::MethodCall(method_call)) => {
```

Comment:
Are method calls only used for `Future::await`, if so, we could probably do away with them entirely.